db/migration/20220430120300_add_post_counter_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddPostCounterToUsers < ActiveRecord::Migration[7.0]
+    def change
+      add_column :users, :post_counter, :integer
+    end
```

Comment:
post_counter already exists in the users migration file. 

Refer to create_users migration file :point_up_2:

Kindly, remove **AddPostCounterToUsers** migration file.